lib/CLI.js

```diff
@@ -2369,7 +2369,7 @@ function checkDeprecates(conf){
  */
 function prepareAppName(conf){
   if (!conf.name){
-    conf.name = p.basename(conf.script);
+    conf.name = conf.script !== undefined ? p.basename(conf.script) : 'undefined';
```

Comment:
How did you trigger this error? `script` should always be defined IMO, maybe we can throw a ReferenceError if it isn't.